src/content/docs/enterprise/organization-management/settings/auhentication.mdx

```diff
@@ -30,6 +30,16 @@ Configure advanced security settings such as **Allow Signup**, **Two-Factor Auth
 
 Select whether people will be able to join the organization without prior invitation via email or a shareable link. After the registration, admins and project managers will be able to manage organization members' permissions.
 
+#### Only administrators can invite users
```

Comment:
Capitalization inconsistency. It would be nice to align the naming in the UI as well